content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md

```diff
@@ -12,125 +12,152 @@ weight: 50
+behavior of the `RequestedToCapacityRatioResourceAllocation` priority function
+can be controlled by a configuration option called
+`requestedToCapacityRatioArguments`. This argument consists of two parameters
+`shape` and `resources`. The `shape` parameter allows the user to tune the
+function as least requested or most requested based on `utilization` and
+`score` values.  The `resources` parameter consists of `name` of the resource
+to be considered during scoring and `weight` specify the weight of each
+resource.
+
+Below is an example configuration that sets
+`requestedToCapacityRatioArguments` to bin packing behavior for extended
+resources `intel.com/foo` and `intel.com/bar`.
 
 ```json
 {
-    "kind" : "Policy",
-    "apiVersion" : "v1",
-
-    ...
-
-    "priorities" : [
-
-       ...
-
-      {
-        "name": "RequestedToCapacityRatioPriority",
-        "weight": 2,
-        "argument": {
-          "requestedToCapacityRatioArguments": {
-            "shape": [
-              {"utilization": 0, "score": 0},
-              {"utilization": 100, "score": 10}
-            ],
-            "resources": [
-              {"name": "intel.com/foo", "weight": 3},
-              {"name": "intel.com/bar", "weight": 5}
-            ]
-          }
+  "apiVersion" : "v1",
+  "kind" : "Policy",
+  ...
+  "priorities" : [
+     ...
+    {
+      "name": "RequestedToCapacityRatioPriority",
+      "weight": 2,
+      "argument": {
+        "requestedToCapacityRatioArguments": {
+          "shape": [
+            {"utilization": 0, "score": 0},
+            {"utilization": 100, "score": 10}
+          ],
+          "resources": [
+            {"name": "intel.com/foo", "weight": 3},
+            {"name": "intel.com/bar", "weight": 5}
+          ]
         }
       }
-    ],
-  }
+    }
+  ],
+}
 ```
 
 **This feature is disabled by default**
 
-### Tuning RequestedToCapacityRatioResourceAllocation Priority Function
+### Tuning the Priority Function
 
-`shape` is used to specify the behavior of the `RequestedToCapacityRatioPriority` function.
+`shape` is used to specify the behavior of the
+`RequestedToCapacityRatioPriority` function.
 
 ```yaml
- {"utilization": 0, "score": 0},
- {"utilization": 100, "score": 10}
+shape:
+ - utilization: 0
+   score: 0
+ - utilization: 100
+   score: 10
 ```
 
-The above arguments give the node a score of 0 if utilization is 0% and 10 for utilization 100%, thus enabling bin packing behavior. To enable least requested the score value must be reversed as follows.
+The above arguments give the node a `score` of 0 if `utilization` is 0% and 10 for
+`utilization` 100%, thus enabling bin packing behavior. To enable least
+requested the score value must be reversed as follows.
 
 ```yaml
- {"utilization": 0, "score": 100},
- {"utilization": 100, "score": 0}
+shape:
+  - utilization: 0
+    score: 100
+  - utilization: 100
+    score: 0
 ```
 
 `resources` is an optional parameter which by defaults is set to:
```

Comment:
Optional extra fix:
```suggestion
`resources` is an optional parameter which by defaults to:
``

Reply:
Done.

Reply:
D'oh, I should have suggested:
```markdown
`resources` is an optional parameter which defaults to:
```
(sorry)

Reply:
never mind, I figured that out.
have to push again to remove a file that doesn't belong to this PR.